### PR TITLE
Expose DebugKit autoRun option

### PR DESCRIPTION
**Pull request by inoas**
... so that people may disable it.

### CODE REVIEW

**Comment by ravage84** [comment]
Refers: https://github.com/cakephp/debug_kit/wiki/Configuration#autorun

I had to search quite a bit to find the documentation on this. I guess the documentation for this could be a bit more visible...

File: config/bootstrap.php [hunk 1]
```diff
@@ -216,7 +216,8 @@
 /*
  * Only try to load DebugKit in development mode
  * Debug Kit should not be installed on a production system
+ * You may also set autoRun to false and append ?debug=true to your URLs to toggle DebugKit on.
  */
 if (Configure::read('debug')) {
-    Plugin::load('DebugKit', ['bootstrap' => true]);
+    Plugin::load('DebugKit', ['bootstrap' => true, 'autoRun' => true]);
```
Comment (markstory) [inline config/bootstrap.php]: I don't think this actually does anything. Instead I think you want autoRun set in Configure.

**Comment by markstory** [comment]
I don't think there is an `autorun` configuration value in 3.x either.

**Comment by inoas** [comment]
You are right, I have no idea why I thought it was working - I did test it - but maybe I overlooked the small icon bottom right.

**Comment by ravage84** [comment]
@markstory I propose to remvoe the Wiki that possibly lead to this confusion in cakephp/debug_kit/issues/532

**Pull request closed without merging**